pages/reference/base-images/base-images.md

```diff
@@ -17,15 +17,15 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
   - amd64
   - i386
 - Multiple Distributions:
-  - [Debian](https://www.debian.org/): jessie (8), stretch (9), buster (10), bullseye (11), and sid
-  - [Alpine](https://alpinelinux.org/): 3.10, 3.11, 3.12, 3.13 and edge
-  - [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), cosmic (18.10), disco (19.04), eoan (19.10) and focal (20.04)
-  - [Fedora](https://getfedora.org/): 30, 31, 32, 33 and 34
+  - [Debian](https://www.debian.org/): stretch (9), buster (10), bullseye (11), bookworm and sid
+  - [Alpine](https://alpinelinux.org/): 3.11, 3.12, 3.13, 3.14 and edge
+  - [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), focal (20.04), hirsute and impish
```

Comment:
Should this mention which release-numbers `hirsute` and `impish` refer to?

Reply:
@lurch updated, thanks for the review :)